Sources/App/Settings/SettingsDetailViewController.swift

```diff
@@ -454,7 +453,26 @@ class SettingsDetailViewController: HAFormViewController, TypedRowControllerType
                         $0.disabled = true
                     },
                 ], getter: {
```

Comment:
you'll need a [weak self] here too

Reply:
Not needed anymore after my last commit.